components/nav/nav.tsx

```diff
@@ -0,0 +1,117 @@
+
+  return (
+    <header className="relative z-10 border-b border-accent-8">
+      <Container width="bleed">
+        <nav className="navWrapper flex justify-between items-center py-3 xl:p-0">
+          <Link href="/" className="flex items-center no-underline">
+            <Logo />
+            <span className="text-base text-foreground">
+              {SiteConfig.author.name}
+            </span>
+          </Link>
+
+          {/* TODO: make the links configurable */}
+          <div className="relative">
+            {mounted && (
+              <div className="flex gap-4 lg:hidden">
+                <button
+                  onClick={() => setTheme(theme === 'dark' ? 'light' : 'dark')}
+                  className="transition-colors text-accent-4 hover:text-accent-1"
+                >
+                  {theme === 'dark' ? <Moon /> : <Sun />}
+                </button>
+                <Hamburger />
+              </div>
+            )}
+
+            <nav className="hidden list-none md:flex md:items-center">
+              <NavItem dataTestId="workPageLink" title="Works" href="/works" />
+              <NavItem
+                dataTestId="articlesPageLink"
+                title="Articles"
+                href="/articles"
+              />
+              <NavItem dataTestId="notesPageLink" title="Notes" href="/notes" />
+              <NavItem dataTestId="aboutPageLink" title="About" href="/about" />
+
+              <NavItem
+                dataTestId="rssPageLink"
+                title={<Rss className="text-foreground" />}
+                href={rssLink}
+                external
+              />
+              <div>
+                {mounted ? (
+                  <button
+                    className="p-4 border-l border-accent-8 text-accent-4 hover:text-accent-1"
+                    onClick={() =>
+                      setTheme(theme === 'dark' ? 'light' : 'dark')
+                    }
+                  >
+                    {theme === 'dark' ? <Moon /> : <Sun />}
+                  </button>
+                ) : (
+                  <div className="p-4 border-l border-accent-8 text-accent-6">
+                    <Loader className="animate-spin" />
+                  </div>
+                )}
+              </div>
+            </nav>
+          </div>
+        </nav>
+      </Container>
+    </header>
+  );
+}
+
+function NavItem({
+  title,
+  href,
+  dataTestId,
+  external = false,
+}: {
+  title: string | ReactNode;
+  href: string;
+  dataTestId?: string;
+  external?: boolean;
+}) {
+  const classes =
+    'border-l border-accent-8 text-accent-4 hover:text-accent-1 p-4 uppercase';
+
+  if (external) {
+    return (
+      <a
+        data-test-id={dataTestId}
+        href={href}
+        className={classes}
+        target="_blank"
+        rel="noreferrer noopener"
+      >
+        {title}
+      </a>
+    );
+  }
+
+  return (
+    <Link data-test-id={dataTestId} href={href} className={classes}>
```

Comment:
Add `data-test-id` for Playwright testing

Reply:
Can this be done differently without a data-test-id?

Reply:
Removed all data-test-ids 

[24018f3](https://github.com/upleveled/next-starter-peacock/pull/156/commits/24018f374b13b8104d6c67a18053d674b6218537)